test-apps/appui-test-app/standalone/src/frontend/appui/frontstages/LocalFileStage.tsx

```diff
@@ -4,23 +4,22 @@
 *--------------------------------------------------------------------------------------------*/
```

Comment:
This file only have import order changes, I suggest reverting the changes so we are not searching for "what" was changed later on.

Reply:
Reverted.